html/controls/rulesets.js

```diff
@@ -235,50 +235,45 @@ function displayRuleset(id) {
 }
 
 function New() {
-	var o = {
-		parent: $("#new_parent").val(),
-		name: $("#new_name").val()
-	};
 	// Note what rulesets we know about so we can determine
 	// the id of the new one when we get an update.
 	knownRulesets = {};
 	$.each(rulesets, function(id) {
 		knownRulesets[id] = true;
 	});
+	var uuid;
+	do {
+		uuid = 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, function(c) {var r = Math.random()*16|0,v=c=='x'?r:r&0x3|0x8;return v.toString(16);}).toUpperCase();
```

Comment:
Apparently Javascript's RNG isn't the best, we probably want to add in the current time here